doc/tutorials/specification.md

```diff
@@ -201,8 +204,12 @@ kdb get /sw/org/app/\#0/current/server/port
 Try to set the port to `123456` now.
 
 ```sh
-kdb set /sw/org/app/\#0/current/server/port 123456
-# STDERR: Port 123456 on key /server/port was not within 0 - 65535
+kdb set user:/sw/org/app/\#0/current/server/port 123456
+ Sorry, 1 warning was issued ;(
```

Comment:
This is not shell recorder syntax. Please add the tutorial to tests/shell/shell_recorder/tutorial_wrapper/CMakeLists.txt so that the tutorial actually gets validated. See also https://github.com/ElektraInitiative/libelektra/tree/master/tests/shell/shell_recorder/tutorial_wrapper

Reply:
I'll take a look at it :)